app/helpers/layout_helper.rb

```diff
@@ -323,6 +323,12 @@ def trunc_with_tooltip(text, length = 32)
     content_tag(:span, truncate(text, :length => length), options).html_safe
   end
 
+  def text_with_tooltip(tooltip_text, text = null, length = 32)
```

Comment:
Did you see @orrabin's earlier comment asking if this can be combined with trunc_with_tooltip somehow, since they're similar?